test/JDBC/expected/Test-sp_helpdbfixedrole-vu-verify.out

```diff
@@ -0,0 +1,93 @@
+EXEC sp_helpdbfixedrole
```

Comment:
For sure, and even if we were doing this within a function this wouldn't lead to any issues since there's no longer any local variables in the implementation.